server/integration/testsuite/src/test/java/org/infinispan/persistence/cluster/MyCustomCacheStore.java

```diff
@@ -2,57 +2,65 @@
 
 import org.infinispan.commons.configuration.ConfiguredBy;
 import org.infinispan.marshall.core.MarshalledEntry;
-import org.infinispan.persistence.spi.PersistenceException;
 import org.infinispan.persistence.spi.ExternalStore;
 import org.infinispan.persistence.spi.InitializationContext;
-import org.infinispan.util.logging.Log;
-import org.infinispan.util.logging.LogFactory;
+import org.infinispan.persistence.spi.PersistenceException;
+
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.fail;
 
 /**
  * Test cache store, which never store/loads anything (= returns null for all keys).
  *
- *  @author Jakub Markos
+ * @author Jakub Markos
  */
 @ConfiguredBy(MyCustomCacheStoreConfiguration.class)
 public class MyCustomCacheStore implements ExternalStore {
 
-    private static final Log log = LogFactory.getLog(MyCustomCacheStore.class);
+   private MyCustomCacheStoreConfiguration config;
 
-    private MyCustomCacheStoreConfiguration config;
+   public MyCustomCacheStore() {
```

Comment:
To be honest - I'm not sure that is the best approach... I can obviously get a comment from both sides on this module - use 4 spaces or use proper formatter :( Maybe we should configure Checkstyle for Infinispan to avoid this kind of problems in the future?

BTW - does this PR look correctly from conceptual point of view? I'm going to need it tomorrow.